cpp/daal/src/algorithms/dtrees/forest/classification/df_classification_train_dense_default_impl.i

```diff
@@ -50,17 +50,15 @@ namespace training
 namespace internal
 {
 //////////////////////////////////////////////////////////////////////////////////////////
-// UnorderedRespHelper
+// UnorderedRespHelperBest class for best splitting classification
 //////////////////////////////////////////////////////////////////////////////////////////
```

Comment:
It would be nice to write somewhere here a descriptive comment that describes the overall *Helper classes hierarchy.

Reply:
Will do, good idea.